js/actions.js

```diff
@@ -1,7 +1,29 @@
+// Selectors for search button
+//TODO - change id names HTML dependant
+var searchButton = $("#searchBtn");
+var searchInput = $("#searchValue");
 
-// !!! remove before writing actual code !!!
-console.log(`
-||||||||||||||||||||||||||||||||||||||||||||||||
-| It's not a bug; it's an undocumented feature |
-||||||||||||||||||||||||||||||||||||||||||||||||`
-)
\ No newline at end of file
+//On click function for search button
+searchButton.on("click", function(event){
+    event.preventDefault();
+    var userInput = searchInput.val().trim();
+//Tasty API Query one - list of meals
+    const settings = {
+        "async": true,
+        "crossDomain": true,
+        "url": "https://tasty.p.rapidapi.com/recipes/list?from=0&size=8&q=" + userInput,
```

Comment:
This is ok, but for reference we could also write it this way:
"url": `https://tasty.p.rapidapi.com/recipes/list?from=0&size=8&q=${userInput}`,
It looks a bit cleaner that way and you save on extra spaces etc.

Reply:
Re "size" property, have a look at this:
![image](https://user-images.githubusercontent.com/41340032/216380570-67b4d48a-1885-4d6d-a44a-fbbf66868d73.png)
then this:
![image](https://user-images.githubusercontent.com/41340032/216380763-e9a11612-c684-497f-b7fd-5814b673ba50.png)

The latter will still return all (28) recipes, but now you don't have extra calls to the API and overhead of consecutive levels within result.